shell/client/admin/user-details.html

```diff
@@ -285,6 +285,36 @@
     <h2>Identities</h2>
     {{> newAdminUserDetailsIdentityTable account=account }}
 
+    <h2>Status</h2>
+
+    <div style="display: flex; flex-direction: row;">
+      Active
+    </div>
+    <form class="standard-form">
+      <div class="button-row">
+        <button type="button" name="delete-account">Delete...</button>
+      </div>
+    </form>
+
+    {{#if showDeletePopup}}
+      {{#modalDialogWithBackdrop onDismiss=cancelDelete}}
+        {{#if deleteError}}
+          {{#focusingErrorBox}}
+            {{deleteError}}
+          {{/focusingErrorBox}}
+        {{/if}}
+
+        <p>Are you sure? This operation is permanent and irreversible.</p>
```

Comment:
Can we adjust this copy?  "operation" seems needlessly technical, and yet we may also want to explain precise *what* will happen when this account is deleted.  Maybe:

```html
<h2>Delete {{guessUserName}}</h2>
<p>Deleting this user account will:</p>
<ul>
  <li>Permanently delete any grains created by {{guessUserName}}</li>
  <li>Revoke any sharing links created by {{guessUserName}}</li>
  <li>Remove {{guessUserName}} from all users' contact lists</li>
  <li>Delete all identities owned by this account (except identities associated with another account)</li>
</ul>
<p><strong>Account deletion is permanent and irreversible.</strong>  Please be certain.</p>
```